src/Firebase.cpp

```diff
@@ -53,11 +61,45 @@ FirebaseGet Firebase::get(const String& path) {
   return FirebaseGet(host_, auth_, path, &http_);
 }
 
+FirebaseSet Firebase::set(const String& path, int value) {
```

Comment:
Yes, but then we won't be able to really generate typed documentation for them, I prefered templating the internal helper: `jsonEncode`.